### ISSUE TITLE
Barking cough

### ISSUE THREAD

**Issue by johnwjudkins** [obophenotype/human-phenotype-ontology#5894]
**Preferred term label:**
Barking cough

**Synonyms**
Honking cough

**Definition (free text, please give PubMed ID)**
A cough that has a characteristic sound similar to the barking of a seal.
PMC2080754

**Parent term (use hpo.jax.org/app)**
Cough HP:0012735

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Croup

**Your nano-attribution (ORCID)**
0000-0001-6595-0902

**Comment by pnrobinson** [comment]
Thanks! Added as HP:0033121
